products/cloudidentity/api.yaml

```diff
@@ -15,6 +15,9 @@
 name: CloudIdentity
 display_name: Cloud Identity
 versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://cloudidentity.googleapis.com/v1beta1/
   - !ruby/object:Api::Product::Version
```

Comment:
This seems wrong

Reply:
what seems wrong about it?

Reply:
ah I put in the wrong url thanks for catching!

Reply:
Fixed in 69e31119bce49986d1671ed0807781ac98ab7d66